src/main/java/it/gov/pagopa/pu/debtpositions/mapper/InstallmentMapper.java

```diff
@@ -74,5 +74,33 @@ public InstallmentDTO mapToDto(InstallmentNoPII installment) {
       .build();
   }
 
+  public InstallmentDTO mapToDto(Installment installment){
+    return InstallmentDTO.builder()
+      .installmentId(installment.getInstallmentId())
+      .paymentOptionId(installment.getPaymentOptionId())
+      .status(installment.getStatus())
+      .iupdPagopa(installment.getIupdPagopa())
+      .iud(installment.getIud())
+      .iuv(installment.getIuv())
+      .iur(installment.getIur())
+      .iuf(installment.getIuf())
+      .nav(installment.getNav())
+      .dueDate(installment.getDueDate())
+      .paymentTypeCode(installment.getPaymentTypeCode())
+      .amountCents(installment.getAmountCents())
+      .notificationFeeCents(installment.getNotificationFeeCents())
+      .remittanceInformation(installment.getRemittanceInformation())
+      .humanFriendlyRemittanceInformation(installment.getHumanFriendlyRemittanceInformation())
+      .balance(installment.getBalance())
+      .legacyPaymentMetadata(installment.getLegacyPaymentMetadata())
+      .debtor(null) //TODO P4ADEV-2028
```

Comment:
why don't you resolve just calling the get?
```suggestion
      .debtor(installment.getDebtor())
```

Reply:
due to this error:
![image](https://github.com/user-attachments/assets/112e6e5c-fa89-409a-ba0f-586d1a092bed)
It's up to who will implement task 2028 to make the mapper Person->PersonDTO, and the corresponding unit tests